Source/Core/VideoCommon/ShaderGenCommon.cpp

```diff
@@ -76,7 +78,7 @@ std::string GetDiskShaderCacheFileName(APIType api_type, const char* type, bool
   {
     // We're using 21 bits, so 6 hex characters.
     ShaderHostConfig host_config = ShaderHostConfig::GetCurrent();
-    filename += StringFromFormat("-%06X", host_config.bits);
+    filename += fmt::format("{:<06X}", host_config.bits);
```

Comment:
Is this really the same as the previous code? The previous one had a dash followed by a 6-digit hex number; but the open angle bracket is just alignment within the available space (as far as I can tell from the `fmt` docs). I'm guessing (without having run the code) that this is now missing the dash.